yax/artifacts/informative_alignment_data.py

```diff
@@ -0,0 +1,21 @@
+from yax.state.type.artifact import Artifact
+from yax.shared.utilities import taxidtool
+
+
+class InformativeAlignmentData(Artifact):
+    def __init__(self, completed):
+        self.tax_id_data = None
+        self.inclusion_tree = None
+        self.gis_to_taxids = None
+        self.sequences_input_fp = None
+        self.truncation_level = None
+
+        if completed:
+            pass
```

Comment:
actually, creating a file which stores the information in a particular (even custom) format is entirely the point of the artifact. It's the `.seqs` and `.tree` files which don't belong here. Artifacts are for complete representations of answers.